### PR TITLE
header

### PR DESCRIPTION
**Pull request by jigmeloday**

### CODE REVIEW

File: apps/know-me/src/styles/_utilities.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  }
+}
+
+.border{
+  &--input fieldset {
+    border-radius: 8px;
+  }
+}
+
+.custom-search-input {
```
Comment (sangayt1997) [inline apps/know-me/src/styles/_utilities.scss]: please try to have _overwrite.scss under styles which you can overwrite styling for all the component

Comment (jigmeloday) [reply apps/know-me/src/styles/_utilities.scss]: oh make different style sheet mo?

Comment (sangayt1997) [reply apps/know-me/src/styles/_utilities.scss]: yup like _overwrite.scss but this overwrite styling have to be affected to whole particular components and incase you want to have those affect in one UI you can do it in that UI component stylesheet

File: libs/shared/src/lib/components/header/header.tsx [hunk 2]
```diff
@@ -1,37 +1,72 @@
...
+              </Badge>
+            <Box pl='22px'>
+              <AccountCircleOutlinedIcon fontSize='large' />
             </Box>
           </Grid>
         </Grid>
-        <Grid item container px='28px'>
-          hi
+        <Box px='28px'>
+          <Divider />
```
Comment (sangayt1997) [inline libs/shared/src/lib/components/header/header.tsx:58]: why can't you use px='28px' in Divider component.

File: libs/shared/src/lib/theme/theme.ts [hunk 3]
```diff
@@ -4,14 +4,14 @@ export const theme = ( mode?: PaletteMode ): Theme => createTheme({
...
     primary: {
-      main: '#43c48e',
-      light: '#7bd6b0',
+      main: '#5fa800',
+      light: 'rgba(95,168,0,0.6)',
       dark: '#287655',
     },
     success: {
       main: '#00C256',
       light: 'rgba(0, 178, 79, 0.12)',
```
Comment (sangayt1997) [inline libs/shared/src/lib/theme/theme.ts:13]: did you use material theming generator to generate the color for success based on main variants?

File: libs/shared/src/lib/components/header/header.tsx (hunk 2 above)
Comment (jigmeloday) [reply libs/shared/src/lib/components/header/header.tsx:58]: divider ga sho mala 
Should i make custom divider?

File: libs/shared/src/lib/theme/theme.ts (hunk 3 above)
Comment (jigmeloday) [reply libs/shared/src/lib/theme/theme.ts:13]: this i will redo after

**Review by sangayt1997 — APPROVED** [review]

**Pull request merged**